pylint/utils/utils.py

```diff
@@ -17,12 +17,32 @@
 import sys
 import textwrap
 import tokenize
+from typing import TYPE_CHECKING, Any, List, Optional, Pattern, Tuple, Union, overload
 
 from astroid import Module, modutils
 
 from pylint.constants import PY_EXTS
 
+if TYPE_CHECKING:
+    from typing import Literal
+
+    from pylint.checkers.base_checker import BaseChecker
+
 DEFAULT_LINE_LENGTH = 79
+GLOBAL_OPTION_BOOL = Union[
```

Comment:
Maybe add a small comment before to mention that these are for `get_global_option`

Reply:
I guess this also means you agree with keeping them here instead of in `pylint/typing`?

Reply:
Yeah. There isn't really a need to import them anywhere else as they are only required for the overloads of `get_global_option`.